nginx.conf

```diff
@@ -27,6 +27,27 @@ http {
     listen <%= ENV["PORT"] %>;
     server_name localhost;
 
+    # Note: .+ instead of .*
+    # this is to allow /data/ to pass through untouched
+
+    location ~* ^/data/(.+) {
+
+      set $url_full         '$1';
+
+      proxy_http_version     1.1;
+      proxy_set_header       Authorization '';
+      proxy_hide_header      x-amz-id-2;
+      proxy_hide_header      x-amz-request-id;
+      proxy_hide_header      Set-Cookie;
+      proxy_ignore_headers   "Set-Cookie";
+      proxy_buffering        off;
+      proxy_intercept_errors on;
+
+      resolver               8.8.8.8 valid=300s;
+
+      proxy_pass             https://s3-us-gov-west-1.amazonaws.com/cg-ff27f612-cd16-4a76-94f8-628986af2158/data/$url_full;
```

Comment:
The bucket URL is currently hardcoded here. This could potentially be moved to an environment variable. However, we are intentionally using the same bucket in both production and staging, so you'd be setting the same environment var in all deployed environments anyway.